include/rcutils/shared_library.h

```diff
@@ -130,6 +130,16 @@ RCUTILS_WARN_UNUSED
 rcutils_ret_t
 rcutils_unload_shared_library(rcutils_shared_library_t * lib);
 
+/// check if the library is loaded
+/**
+ * \param[in] lib rcutils_shared_library_t  to check
+ * \return true if library is loaded, false otherwise
```

Comment:
I am pretty sure that there are cases where this function will return `false` where the library is actually still loaded. This function only determines if "unload" has been called on the current shard library handle. It could very well be that a second shared library handle is still open and therefore the library being loaded.

Reply:
how should I implement this function ? can you share your thoughts?

Reply:
I don't think this can be resolved in the implementation. I think clarifying this in the docblock is the only way possible.